Road/source/utils.py

```diff
@@ -5,3 +5,35 @@ def function_name(parameter_name:type)->return_type:
     local_variable=0
     return
 """
+import pandas as pd
+
+import csv
+
+from Road import logger
+
+
+def read_n_from_file(
+    number_of_messages: int=5, file_path: str = "Road/data/interim/feature_selected.csv"
+) -> pd.DataFrame:
+    """
+    Reads n messages from the end of a CSV file and returns them as a pandas DataFrame.
+
+    Args:
+        number_of_messages (int): The number of messages to read from the file.
```

Comment:
`number_of_message` is optional arg

Reply:
i will add the default value to the comment